document/core/binary/modules.rst

```diff
@@ -512,7 +514,7 @@ The :math:`\MEMORYINIT` and :math:`\DATADROP` instructions can only be used if t
      \Bcustomsec^\ast \\ &&&
      \X{code}^n{:\,}\Bcodesec \\ &&&
      \Bcustomsec^\ast \\ &&&
-     \data^{\X{m'}}{:\,}\Bdatasec \\ &&&
+     \data^m{:\,}\Bdatasec \\ &&&
```

Comment:
Is there syntax for this elsewhere? It seems odd to me to take `m^?` and remove the `^?` -- I suppose I'd guess that it would produce `0`, but it would be nice to mention that somewhere.

Reply:
I seem to remember that I've already used this kind of trick elsewhere, but I cannot find it right now. I added an additional bullet to make this kind of thing explicit in the conventions.